WalletWasabi.Fluent/Views/Wallets/Receive/ReceiveAddressView.axaml

```diff
@@ -30,11 +30,11 @@
                               ClickIcon="{StaticResource copy_confirmed}"
                               ExecuteOnOpen="{Binding IsAutoCopyEnabled}" />
           </StackPanel>
-          <c:PrivacyContentControl FontSize="{StaticResource FontSizeH5}" NumberOfPrivacyChars="38">
+          <c:PrivacyContentControl FontSize="{StaticResource FontSizeH5}" NumberOfPrivacyChars="40" MinHeight="25" VerticalContentAlignment="Center">
```

Comment:
The "pushing" effect that the `MinHeight` fixes is caused because of the font difference between the privacy mask and the original text.

It is even a bug, the `PrivacyContentControl` has a style defined for monospace stuff, I don't understand why is it added just for the TextBlock.

I would just simply use that style instead of the MinHeight and also it can be simplified once it is done:
```xml
<c:PrivacyContentControl Classes="monoSpaced"
                         FontSize="{StaticResource FontSizeH5}"
                         NumberOfPrivacyChars="40"
                         VerticalContentAlignment="Center"
                         Content="{Binding Address}" />
```

Reply:
Ok, I see it is problem everywhere. You can ignore it if you want.

Reply:
@soosr I would rather not change any fonts or anything to prevent my PRs from becoming controversial and thrown into the endless void of oblivion for all eternity